workflow/search-filter.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Search & Filter
```

Comment:
We need two more sections in this doc:
1. Filtering using attributes (the left side bar)
2. Using the searchbox

The search box section should be pretty extensive on what type of functions are supported in the search bar and some examples of how to use it.

Reply:
DD does the search syntax/examples well https://docs.datadoghq.com/logs/explorer/search_syntax/